code/game/objects/items/weapons/melee/energy.dm

```diff
@@ -8,6 +8,10 @@
 	armor_penetration = 50
 	flags = NOBLOODY
 	can_embed = 0//No embedding pls
+	var/base_reflectchance = 40

```

Comment:
40% is absurd. make this 10% or less pls
energy swords were already crazy powerful before